src/local/butler/scripts/setup.py

```diff
@@ -142,7 +142,7 @@ def __init__(self):
 avg(t.average_exec_per_sec) as avg_exec_per_sec,
 avg(t.fuzzing_time_percent) as fuzzing_time_percent,
 sum(t.new_units_added) as new_tests_added,
-sum(t.merge_new_features) as new_cov_features,
+sum(t.new_cov_features) as new_cov_features,
```

Comment:
maybe just "features" cov is a bit redundant

Reply:
sure, will do.